Header header
uint64 trackingId
float32 jawOpen
float32 lipPucker
float32 jawSlideRight
float32 lipStretcherRight
float32 lipStretcherLeft
float32 lipCornerPullerLeft
float32 lipCornerPullerRight
float32 lipCornerDepressorLeft
float32 lipCornerDepressorRight
float32 leftCheekPuff
float32 rightCheekPuff
float32 leftEyeClosed
float32 rightEyeClosed
float32 leftEyebrowLowerer
float32 rightEyebrowLowerer
float32 lowerLipDepressorLeft
float32 lowerLipDepressorRight
geometry_msgs/Quaternion faceOrientation
geometry_msgs/Point headPivotPoint
